server/adapters/restore.js

```diff
@@ -0,0 +1,49 @@
+//
+// Adapter Restore
+//
+module.exports = function initRestoreAdapters(models, adapters) {
+
+  //
+  // Create new restore instance
+  //
+  adapters.createRestore = function (userId, saveId) {
+    models.restore.create({
+      userId: userId,
+      saveId: saveId,
+      execDate: new Date(),
+    });
+  };
+
+  //
+  // Search in the database a restore instance with id = restoreId
+  // Update boolean isStart
+  //
+  adapters.restoreIsStart = function (restoreId) {
```

Comment:
By convention, a function containing "Is" should return a boolean (from my point of view). So this function and the ones below should be named "restoreStart" to avoid any confusion.